v3/cypress/support/elements/component-elements.ts

```diff
@@ -70,6 +70,12 @@ export const ComponentElements = {
   getIconFromToolShelf(component: string) {
     return toolbar.getToolShelfIcon(component)
   },
+  clickIconFromToolShelf(component: string) {
+    this.getIconFromToolShelf(component).click()
```

Comment:
I'd be curious where you read that because I view it as a code smell. The user can't `{ force: true }` their clicks, so it has the potential to cause cypress tests to work in situations in which the feature wouldn't work for users.